src/soundbase.cpp

```diff
@@ -166,30 +166,38 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ).arg ( strSystemDriverTechniqueName ) + "</b><br>";
+            QString sErrorMessage = "<b>";
             if ( lNumDevs > 1 )
             {
                 // only output multiple messages if more then one are necessary.
-                sErrorMessage += "<br>" + tr ( "These are all the available drivers with error messages:" ) + "<ul>";
+                sErrorMessage += tr ( "Sound driver errors (using %1) found:</b><br>"
+                                      "None of your sound devices are compatible in their current state." )
+                                     .arg ( strSystemDriverTechniqueName );
+
                 for ( int i = 0; i < lNumDevs; i++ )
                 {
-                    sErrorMessage += "<li><b>" + GetDeviceName ( i ) + "</b>: " + vsErrorList[i] + "</li>";
+                    sErrorMessage += "<b>" + GetDeviceName ( i ) + "</b>: " + vsErrorList[i] + "<br>";
                 }
-                sErrorMessage += "</ul>";
             }
             else
             {
-                 sErrorMessage += "<b>" + GetDeviceName ( 0 ) + "</b>: " + vsErrorList[0] ;
+                sErrorMessage += tr ( "Couldn't initialize audio driver (using %1)</b><br>"
```

Comment:
Line 173:
> `"Sound driver errors (using %1) found:</b><br>"`

Line 184:
> `"Couldn't initialize audio driver (using %1)</b><br>"`

Use the same string in both places.  Ideally the less technical one.  The first doesn't need to end with ":" as it no longer introduces a list.

Maybe
> `"Could not find a sound driver (using %1) that could be used.</b><br>"`

(and do away entirely with `"Your sound device is not compatible in its current state."` as it would then be somewhat redundant).